Restructure the section on how to administer composite db (#1586)
neo4j/docs-operations: modules/ROOT/pages/database-administration/composite-databases/list-composite-databases.adoc

:description: This page describes how to list composite databases.
[role=enterprise-edition not-on-aura]
[[composite-databases-list]]

> = Listing composite databases

Reviewer: I wonder if we should make all headings consistent - using the `-ing` or `infinitive` form.
```suggestion
= List composite databases
```

////
[source, cypher, role=test-setup]
----
CREATE COMPOSITE DATABASE `library`;
CREATE DATABASE `sci-fi`;
CREATE ALIAS `library`.`sci-fi` FOR DATABASE `sci-fi`;
----
////

> The `type` column returned by the `SHOW DATABASES` command will display which databases are composite databases.

Reviewer: ```suggestion
The `type` column returned by the `SHOW DATABASES` command displays which databases are composite.
```

+----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
| "library" | "composite" | []                 | "read-only"  | "localhost:7687" | NULL      | FALSE  | "online"        | "online"      | ""            | FALSE   | FALSE | ["library.sci-fi"] |
| "neo4j"   | "standard"  | []                 | "read-write" | "localhost:7687" | "primary" | TRUE   | "online"        | "online"      | ""            | TRUE    | TRUE  | []                 |
| "sci-fi"  | "standard"  | ["library.sci-fi"] | "read-write" | "localhost:7687" | "primary" | TRUE   | "online"        | "online"      | ""            | FALSE   | FALSE | []                 |
| "system"  | "system"    | []                 | "read-write" | "localhost:7687" | "primary" | TRUE   | "online"        | "online"      | ""            | FALSE   | FALSE | []                 |
+----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
----

> For a description of all the returned column of this command, ways in which the `SHOW DATABASE` command can be filtered, and details about the privileges required for the command, see xref:database-administration/standard-databases/listing-databases.adoc#manage-databases-list[listing standard databases].

Reviewer: Same as above.
```suggestion
For a description of all the returned columns of this command, ways in which the `SHOW DATABASE` command can be filtered, and details about the privileges required for the command, see xref:database-administration/standard-databases/listing-databases.adoc#manage-databases-list[List standard databases].
```
